mpas_analysis/test/test_mpas_config_parser.py

```diff
@@ -0,0 +1,65 @@
+"""
+Unit test infrastructure for MpasAnalysisConfigParser
+
+Xylar Asay-Davis
+12/05/2016
+"""
+
+import pytest
+from mpas_analysis.test import TestCase, loaddatadir
+from mpas_analysis.configuration.MpasAnalysisConfigParser \
+    import MpasAnalysisConfigParser
+
+
+@pytest.mark.usefixtures("loaddatadir")
+class TestMPASAnalysisConfigParser(TestCase):
+    def setup_config(self):
+        configPath = self.datadir.join('config.analysis')
+        self.config = MpasAnalysisConfigParser()
+        self.config.read(str(configPath))
+
+    def check_container(self, container, container_type, item_type):
+        assert isinstance(container, container_type)
+        for item in container:
+            assert isinstance(item, item_type)
+
+    def test_read_config(self):
+        self.setup_config()
+
+        # check config.get(...)
```

Comment:
NBD: probably don't need a comment like this that states essentially the command of the next line of code.  This is true of all comments in this method.